### PR TITLE
Add sun-shield favicon (#31)

### LINKED ISSUE #31: Add favicon
Maybe a sunrise?

### PR DESCRIPTION
**Pull request by cajunluke**
Closes #31 

These images were generated from my half-day-favicon.svg by https://realfavicongenerator.net/, I'm not sure we _need_ all of them, but they're nice to have, maybe?

### CODE REVIEW

**Comment by dispatchrabbi** [comment]
In general these look good, except:
- I think the site manifest and browserconfig should probably be at the root, not in images/
- I also think it'd be neat if we could keep the SVG as the normal favicon instead of having a converted PNG or ICO. Should work for [all desktop browsers](https://en.wikipedia.org/wiki/Favicon#How_to_use) (except Safari, which does the mask-icon thing)

File: layout/index.hbs [hunk 1]
```diff
@@ -5,12 +5,23 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <link rel="icon" type="image/svg+xml" href="./images/favicon.svg" />
+    <!-- browsers -->
+    <link rel="icon" type="image/svg+xml" href="/images/half-day-favicon.svg">
```
Comment (dispatchrabbi) [inline layout/index.hbs:9]: You should make these `./images/whatever` - helps when dev’ing and the root of your dev webserver isn’t dist/

**Comment by dispatchrabbi** [comment]
Left a comment. Once that’s done, approved.

File: layout/index.hbs (hunk 1 above)
Comment (cajunluke) [reply layout/index.hbs:9]: Got it, will do.

**Pull request merged**